ms-windows/osgeo4w/msvc-env.bat

```diff
@@ -55,6 +55,7 @@ if not exist "%OSGEO4W_ROOT%\bin\o4w_env.bat" (echo o4w_env.bat not found & goto
 call "%OSGEO4W_ROOT%\bin\o4w_env.bat"
 call "%OSGEO4W_ROOT%\bin\py3_env.bat"
 call "%OSGEO4W_ROOT%\bin\qt5_env.bat"
+call "%OSGEO4W_ROOT%\bin\gdal-dev-env.bat"

```

Comment:
grep tells me the only other thing calling this file is [package.cmd](https://github.com/qgis/QGIS/blob/0ada65b210cc57814393d00ec82312e6703940fe/ms-windows/osgeo4w/package.cmd#L37). Will this new line be a problem?

Reply:
yes, the nightlies are build against the proj and nightlies, while the release package are build against the release packages.

Reply:
Requirements:
 - msvc-env.bat shall not impose specific gdal or proj libraries so that it can be used in package.cmd and package-nightly.cmd
 - package.cmd shall pull from nightly builds of gdal and proj libraries
 - package-nightly.cmd shall pull from release gdal and proj libraries in apps\*-dev\bin

Desirement:
 - user can execute exact **Building with Microsoft Visual Studio** workflow to generate and run a working QGIS executable

Rather than moving the call into msvc-env.bat, would it be better to direct the user to `call gdal-dev-env.bat` in the command prompt?